apps/myanmar_calendar/src/components/modals/DayDialog/DayDialog.tsx

```diff
@@ -64,29 +64,29 @@ const DayDialog = ({ onClose, selectedDay }: DayDialogProps) => {
         leaveTo={`${enterMobileMode ? "translate-y-[100%]" : "opacity-0  translate-y-5"}`}
         enter={`ease-out ${enterMobileMode ? "duration-200" : " duration-200"}`}
         leave={`ease-in ${enterMobileMode ? "duration-200" : " duration-150"}`}>
-        <Dialog.Panel className="fixed inset-0 mx-auto mt-auto sm2:my-auto w-full sm2:max-w-[27rem] h-[calc(100%-5rem)] sm2:h-[90%] sm2:max-h-[33rem] transform overflow-hidden rounded-tr-2xl rounded-tl-2xl sm2:rounded-[0.5rem] bg-white text-left align-middle transition-all flex flex-col sm2:border  sm2:border-gray-200 shadow-model">
+        <Dialog.Panel className="fixed inset-0 mx-auto mt-auto sm2:my-auto w-full sm2:max-w-[27rem] h-[calc(100%-5rem)] sm2:h-[90%] sm2:max-h-[33rem] transform overflow-hidden rounded-tr-2xl rounded-tl-2xl sm2:rounded-[0.5rem] bg-cgray-0 text-left align-middle transition-all flex flex-col sm2:border  sm2:border-cgray-200 shadow-model">
           {/* ------ Header ------ */}
-          <div className="h-[3.25rem] sm2:h-[3rem] flex items-center justify-between px-5 sm2:px-3 bg-gray-50 border-b">
+          <div className="h-[3.25rem] sm2:h-[3rem] flex items-center justify-between px-5 sm2:px-3 bg-cgray-50 border-b">
```

Comment:
Convention like cgray did not give us any extra understanding could you please improve naming convention?

Reply:
I think I will go with just gray, maybe that might be sound a bit better, i guess. I meant ColorGray but you are right, it is a bit misleading. I will go with gray and try to fix the boldness!

Reply:
Hey thank you for that.
Please fix and updat.